### ISSUE TITLE
Visual targets should be painted on the arena

### ISSUE THREAD

**Issue by qgeissmann** [gilestrolab/pySolo-Video#21]
It is crucial that we have  **durable** 4mm circles painted on each arena for automatic recognition (see https://github.com/gilestrolab/pySolo-Video/blob/psv-package/src/pysolovideo/tracking/roi_builders.py).

**Comment by ggilestro** [comment]
@PolygonalTree I would reccomend using a casting system for this as well. Recess down the borders of the arena by 0.5mm or so then glue in a circle of a different color so that it will stay flush with the border itself. I would also recess a smal rectangular slot to use as recess for a label containing barcodes for future uses (for instance when we deal with 100+ arenas and we want automatic cataloguing of experiments) see attached screenshot to see what I mean.

![screenshot from 2015-02-04 16 50 39](https://cloud.githubusercontent.com/assets/696901/6044843/fe2c9c7c-ac8d-11e4-91c6-200fa0075f40.png)

**Comment by PolygonalTree** [comment]
That was the original idea, I am still thinking about use something of 0.5mm black that fits in the holes or black paint.

**Comment by PolygonalTree** [comment]
I made some modifications to the arena. @qgeissmann, Are the holes in the right place? 
and @ggilestro  let me know if this is what you want. (space for the barcode is 20mm width by 4mm height)
![captura de pantalla de 2015-02-04 17 54 01](https://cloud.githubusercontent.com/assets/5161395/6046396/b7b2fd66-ac99-11e4-8772-ca87671a3023.png)

**Comment by ggilestro** [comment]
Looks good to me.

**Comment by qgeissmann** [comment]
**Not quite**, My script assumes targets (ABC) are like so:
```
        # B--------------------------C
        # |
        # |
        # |
        # A
```
So, unless it means much more work for you, we may want to keep my way. 

Also, I will need to know is the exact distance between the centre of the targets and the internal edge (i.e. the first ROI).
Cheers,

**Comment by PolygonalTree** [comment]
I am not sure where the little gap is. I assume for your comment that this little gap is under B and C and so therefore I have to change the position of the lower hole. @qgeissmann , please confirm that to made the change.

**Comment by qgeissmann** [comment]
Basically, the lower hole (A) must be on the left.
Then, in the former arena, B and C were not touching the edge of the first tube (gap).
However, ideally, the centre of the targets are all at a same distance `d` from the **edge of nearest tube first tube**. I will need update the algorithm anyway I will just need to know `d`. Does is make sense?

**Comment by PolygonalTree** [comment]
So, Here it is the dimensions.
![captura de pantalla de 2015-02-06 11 28 54](https://cloud.githubusercontent.com/assets/5161395/6078690/655df448-adf3-11e4-90b1-d929105f91bc.png)

**Comment by qgeissmann** [comment]
This is **fantatic**! Cheers. I assume I can close this issue. looking forward to having the new model in my hands though ;).

**Comment by qgeissmann** [comment]
@PolygonalTree, sorry for reopening that, but
From the arena you gave me, I get:
```
        #                            A
        #                            |  
        #                            |
        #                            |
        # C------------------------- B
```
![shot0002](https://cloud.githubusercontent.com/assets/1399684/6135101/b91dfd54-b15d-11e4-92e2-f62dad0aefc8.jpg)

This is dues to the way the camera/pi are put on their vertical support.
Because they are asymmetrical, I cannot simply **rotate the camera 180 degrees**.
I could obviously change the algo, but this orientation is **not** the consensus we have been working with.
Does that make sense?

**Comment by PolygonalTree** [comment]
There is two options:
1. modifiy the pi case to rotate the camera 180 degrees.
2. change the dots to 
`
#A--------------------------B
#
#
#------------------------(1mm gap)
#C

Which do you prefer?

**Comment by qgeissmann** [comment]
I am not quite sure how you come up with the 1mm gap, but it is fine for me (I guess it means less work for you than modifiying the case).

**Comment by qgeissmann** [comment]
Ok, so we are adopting the position **in the picture** as conventional orientation.
@PolygonalTree  can you ensure we update the user protocol/doc (at least, https://github.com/gilestrolab/pySolo-Video/blob/psv-package/misc/arena-guide.svg)

**Comment by qgeissmann** [comment]
fixed by ed6a278a8b220039956b3aac9fbfd61e96d5e17b and e7cf56d3353c890dd70bfef07940573506c8f131 : 
![test](https://cloud.githubusercontent.com/assets/1399684/6150452/7acd089c-b207-11e4-82ef-45b8b6251711.jpg)
